### PR TITLE
[v3i/simd]: Minor update for the V3Interpreter

### PR DESCRIPTION
**Pull request by haoyu-zc**

### CODE REVIEW

File: src/engine/V3Eval.v3 [hunk 1]
```diff
@@ -1090,32 +1090,30 @@ def do_v128_bitmask<T>(a: (u64, u64), bits: byte, bits_log2:byte) -> i32 {
...
+	convert: u64 -> T, extend: (T) -> Tw, convert_to_u: (Tw) -> Uw) -> u64 {
 	var res: u64 = 0u;
 	var shift = byte.view(dest_bits >> 1);	// Compute source shift amount, i.e., the width of the original lane.
-	var iters = byte.view(64 / dest_bits);	// Compute number of iterations. Can be deduced from "shift", put here for clarity.
-	for (i < iters) {
+	var dst_shift = shift << 1;
+	for (i = 0; i < 64; i += dst_shift) {
 		var val = extend(convert(u64.view(src)));
 		src >>= shift;
-		var res_shift = byte.view(shift * 2 * i);	// Compute shift amount for result.
```
Comment (titzer) [inline src/engine/V3Eval.v3:1112]: Can also improve this to a `u6.view`, which will save a another shift check underneath.

**Review by titzer — APPROVED** [review]
lgtm with minor suggestion. Thanks!

File: src/engine/V3Eval.v3 (hunk 1 above)
Comment (haoyu-zc) [reply src/engine/V3Eval.v3:1112]: No prob! Missed this one. Will push a change soon.

Comment (haoyu-zc) [reply src/engine/V3Eval.v3:1112]: Updated:)

**Review by titzer — APPROVED** [review]

**Pull request merged**